_posts/2017-11-20-owasp-ctf.md

```diff
@@ -10,25 +10,14 @@ Présentation de 3 défis du Mini-CTF OWASP. Au menu, des techniques
 d’injection SQL, du contournement de protection de téléversement de fichier et une
 introduction à l’exploitation de unserialize PHP.
 
-Nous avons un nombre limité de places disponibles, [**l'inscription est
-obligatoire**](https://www.eventbrite.ca/e/billets-montrehack-39923111100)
-ce mois-ci **avant dimanche 15:00**.
+Nous avons un nombre limité de places disponibles, **l'inscription est
+obligatoire** ce mois-ci **avant dimanche 15:00**.
 
 ## Outils et expérience requis
 
 * Votre langage de scripting favori (ex.: Python).
 * Un proxy d'interception (ex.: Burp ou Zap).
 
-## Où
-
-OVH
-
-1801 Avenue McGill College #800, Montréal, QC H3A 2N4
-
-## Quand
-
-Lundi le 20 novembre de 18h à 21h (suivi de bières au Bénélux)
-
 ## Comment
```

Comment:
I tend to also remove the "How" since it's repetitive and kind of boring. People looking at the archives would know.